### DISCUSSION TITLE
[feature] Add Icon for Taiko Network

### DISCUSSION THREAD

**Discussion by ablankstory** [reown-com/appkit#2760] [Ideas]
### What problem does this new feature solve?

This request would allow users to better identify the Taiko network when selecting between networks via the w3m-modal.

### Describe the solution you'd like

I'd like to add the official icon for the Taiko network in the network selector modal. Right now it appears as the default network image:

<img width="629" alt="Screenshot 2024-08-27 at 9 53 43 PM" src="https://github.com/user-attachments/assets/8f3f9c56-057d-4a57-bce9-dcc85a5a5115">

Here is a link to their GitHub for the official icons:

https://github.com/taikoxyz/taiko-mono/blob/main/packages/taikoon-ui/static/chains/taiko.svg

If you visit their official Trailblazers site, https://trailblazers.taiko.xyz/, and attempt to switch networks, you can see they've added that same icon for their network:

<img width="379" alt="Screenshot 2024-08-27 at 9 57 24 PM" src="https://github.com/user-attachments/assets/c5dafc6f-b06e-4bcd-be57-c16910cf98a5">

Thanks!

**Comment by glitch-txs** [comment]
ATM you can use this
https://docs.walletconnect.com/appkit/react/core/options#chainimages
